LeaderboardBackend/Authorization/JwtSecurityTokenHandlerSingleton.cs

```diff
@@ -0,0 +1,15 @@
+using System.IdentityModel.Tokens.Jwt;
+
+namespace LeaderboardBackend.Authorization;
+
+/// <summary>
+/// Class that holds a JwtSecurityTokenHandler instance.
+/// </summary>
+/// <remarks>
+/// Singleton definition style taken from https://csharpindepth.com/Articles/Singleton.
+/// </remarks>
+public sealed class JwtSecurityTokenHandlerSingleton
```

Comment:
I think this can be simplified to being a static class and remove the constructor.